packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CirclesController.java

```diff
@@ -67,7 +67,15 @@ boolean onCircleTap(String googleCircleId) {
     if (circleId == null) {
       return false;
     }
-    methodChannel.invokeMethod("circle#onTap", Convert.circleIdToJson(circleId));
+    flutterApi.onCircleTap(
+        circleId,
+        new Messages.VoidResult() {
+          @Override
+          public void success() {}
+
+          @Override
+          public void error(@NonNull Throwable error) {}
+        });
```

Comment:
These are because we don't currently have a way in Pigeon to ignore responses. Longer term I think we'll want to convert this over to event channels though; I'm not sure why maps wasn't written using them in the first place, since a stream of one-way calls without responses are the use cases event channels were written for IIUC.

(I'm not doing that conversion as part of this because we don't have event channel support for Pigeon yet; probably in Q3 sometime.)

Reply:
Now that I think of it, I'm not sure when event channels were added, so this code may just predate them.